aboutMe/AboutMe.md

```diff
@@ -0,0 +1,51 @@
+# Resume
+
+Content
+
+[TOC]
+
+---
+
+## Basic information
+
+### Name
+
+Jamal Guyo
```

Comment:
You can put the name inline to the name title  like this  
 **Name:**  Jamal Guyo

Reply:
resolved